aptos-move/framework/move-stdlib/sources/ascii.move

```diff
@@ -0,0 +1,148 @@
+/// The `ASCII` module defines basic string and char newtypes in Move that verify
```

Comment:
interesting :grimacing:

Reply:
No new move sources should be added here. Aptos does not have ascii any more by intention because we added string. Please do not mix in any design changes or suggestions, this is purely about the native implementations. We can have a discussion about additions separately.

Reply:
> interesting 😬

Not that much. ASCII is legacy and was replaced by string.

Reply:
Removed